src/early_multicellular_stage/editor/CellBodyPlanEditorComponent.cs

```diff
@@ -390,6 +390,13 @@ public override void OnFinishEditing()
         editedSpecies.UpdateNameIfValid(newName);
 
         behaviourEditor.OnFinishEditing();
+
+        foreach (var cellType in Editor.EditedSpecies.CellTypes)
```

Comment:
Looking at the early multicellular species OnEdited method, it seems that is already does RepositionToOrigin for all cell types. So I think this code add here is basically a duplicate operation that is not necessary.

Reply:
Yeah, everything gets repositioned just fine without that, so it wasn't needed.